operations/troubleshooting.md

```
@@ -0,0 +1,290 @@
+
+### Query Timeout Exceeded
+
+**Symptom:** `Query timed out` error.
+
+**Cause:** The query execution exceeded the configured timeout.
+
+**Solutions:**
+
+1. **Check the current timeout settings:**
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)